.github/workflows/ci.yml

```diff
@@ -0,0 +1,86 @@
+name: Alfresco Java SDK CI
+
+on:
+  push:
+    branches:
+      - develop
+      - release/**
+      - dev-**
+  pull_request:
+    branches:
+      - develop
+      - release/**
+      - dev-**
+  workflow_dispatch:
+
+env:
+  MAVEN_PASSWORD: ${{ secrets.NEXUS_PASSWORD }}
+  MAVEN_USERNAME: ${{ secrets.NEXUS_USERNAME }}
+  GITHUB_ACTIONS_DEPLOY_TIMEOUT: 60
+  CONTENT_SERVICE_SECURITY_BASICAUTH_PASSWORD: ${{ secrets.CONTENT_SERVICE_SECURITY_BASICAUTH_PASSWORD }}
+  CONTENT_SERVICE_SECURITY_BASICAUTH_USERNAME: ${{ secrets.CONTENT_SERVICE_SECURITY_BASICAUTH_USERNAME }}
+  CONTENT_SERVICE_URL: ${{ secrets.CONTENT_SERVICE_URL }}
+  ENVIRONMENT_APA_HOST: apadev-apa.envalfresco.com
+  ENVIRONMENT_HOST: apadev.envalfresco.com
+  ENVIRONMENT_APP: simpleapp
+  AGS_VERSION: 14.9
+  ACS_VERSION: 7.1.0.1
+
+
+jobs:
+  pre-commit:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: Alfresco/alfresco-build-tools/.github/actions/pre-commit@v1.35.0
+
+  build_and_verify:
+    name: "Build and Verify"
+    runs-on: ubuntu-latest
+    needs:
+      - pre-commit
+    env:
+      MAVEN_CLI_OPTS: >
+        source:jar -B -q -e -fae -V -DinstallAtEnd=true -U -Ddoclint=none
+        -DaltReleaseDeploymentRepository=alfresco-public::default::https://artifacts.alfresco.com/nexus/content/repositories/releases
+        -DaltSnapshotDeploymentRepository=alfresco-public-snapshots::default::https://artifacts.alfresco.com/nexus/content/repositories/snapshots
```

Comment:
they are probably not needed anymore given that [pom](https://github.com/Alfresco/alfresco-java-sdk/blob/develop/pom.xml#L104) already point those

Reply:
Done